test/test_namedtensor.py

```diff
@@ -82,6 +82,12 @@ def test_has_names(self):
         self.assertTrue(partially_named.has_names())
         self.assertTrue(fully_named.has_names())
 
+    def test_contains(self):
```

Comment:
This shouldn't be in test_namedtensor.py, it should be in `test_torch.py`. We're checking the correctness of `Tensor.__contains__` which isn't related to named tensor functionality.

Reply:
My bad, I think you are right~